eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -286,7 +294,7 @@ jobs:
     displayName: Build
 
   # Don't run tests if overriding RID, we don't support that for now
-  - ${{ if eq(parameters.targetRid, '') }}:
+  - ${{ if eq(parameters.runTests, 'True') }}:
```

Comment:
Please double check that this doesn't disable the testing in the source-build leg. I think it does.

Reply:
It does - I just checked.

Reply:
Knew I forgot something. Will fix.

Reply:
I assume YML ignores case? `true` vs `True`

Reply:
As per the documentation, `eq` performs "Ordinal ignore-case comparison for Strings"